### DISCUSSION TITLE
Change prettify function dynamically?

### DISCUSSION THREAD

**Discussion by munger2k** [IonDen/ion.rangeSlider#763] [General]
Hi, I was wondering if it's possible to change the prettify on the slider once it's been initialized?
I have a slider I'm using to filter a datatable by a height range and want the user to be able to change their display between imperial and metric. 
I'm assuming I can destroy the existing slider and initialize another one but just curious if there is a way to simply change the prettify function of an existing slider.

**Comment by IonDen** [comment]
Hi, prettify function is calling on the fly. So you can move it out and change then you need.

Example: 
```javascript
function myPrettify(n) {
  // do any conditions here
  return n;
}

$("#range").ionRangeSlider({
    type: "double",
    min: 0,
    max: 1000,
    from: 200,
    to: 500,
    prettify: myPrettify
});
```

**Reply by munger2k** [reply]
Thanks a lot for getting back to me, I tired adding the conditions directly into the the prettify function which didn't seem to work but didn't think of creating a separate function for it. Perfect, thanks!
